lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -49,17 +49,12 @@ var DataObservatoryMultipleMeasures = Node.create(TYPE, PARAMS, { cache: true, l
             throw new Error('The number of numerators=' + node.numerators.length +
                 ' does not match the number of numerator_timespans=' + node.numerator_timespans.length);
         }
-    }
+    },
```

Comment:
Removed. In case we want add it -> https://github.com/jshint/jshint/issues/136#issuecomment-197875841